### PR TITLE
attach: bugfixes

### PR DESCRIPTION
**Pull request by brauner**
Signed-off-by: Christian Brauner <christian.brauner@ubuntu.com>

### CODE REVIEW

File: src/lxc/attach.c [hunk 1]
```diff
@@ -66,6 +66,7 @@
 #include "lsm/lsm.h"
 #include "lxclock.h"
 #include "lxcseccomp.h"
+#include "macro.h"
```
Comment (Blub) [inline src/lxc/attach.c]: Should probably be part of the `macro: move macros from attach.c` commit as I see no relation to `rexit()`.

File: src/lxc/macro.h [hunk 2]
```diff
@@ -27,6 +27,7 @@
 #include <linux/loop.h>
 #include <linux/netlink.h>
 #include <linux/rtnetlink.h>
+#include <stdbool.h>
```
Comment (Blub) [inline src/lxc/macro.h]: Is this a leftover? Not required `_exit()` at least, maybe also as part of the `macro...` commit?

**Comment by stgraber** [comment]
needs rebase

**Comment by brauner** [comment]
rebased

**Pull request merged**